iOS14UIKitTour/iOS14UIKitTour/Models/Contributor.swift

```diff
@@ -1,6 +1,14 @@
 
-struct Contributor {
-    let name: String
-    let id: String
-    let repos: [Repository]
+struct Contributor: Codable {
+    let userName: String
+    let avatarURL: String
+    let accountURL: String
+    var contributions: Int?
```

Comment:
Why optional?

Reply:
Because the owner of the repository doesn't have a contributions property

Reply:
We are using this struct for two places: The owner of the repo in the Repositories response, and the contributor

Reply:
I see...